korean-air/src/components/searchPage/SearchHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+import { useNavigate } from "react-router-dom";
+import { IcCloseSmall } from "../../assets";
+interface State {
+  state: string;
+}
+const SearchHeader = (props: State) => {
```

Comment:
interface 사용해서 컴포넌트 props 타입지정해준거 진짜 너무잘했다!! 이렇게만 해주면돼!! 
그리고 만약, 이렇게 넘겨주는 props가 한두개 뿐일 때는 그냥 아래처럼 interface로 분리안하고 바로 사용할 수도 있긴해!! 
이건 개인 취향 ㅎㅎ 
```suggestion
const SearchHeader = ({state} : {state : string}) => {
```

Reply:
헐 훨씬 깔꼼하다!